src/chemistry/pp_trop_strat_mam4_vbs/mo_sim_dat.F90

```diff
@@ -246,14 +246,14 @@ subroutine set_sim_dat
                           1278,1291,1304,1327,1352,1507,1549,1640,1691,1716, &
                           1739,1845,1876,1900,1935,1993,2054,2080 /)
 
-      extfrc_lst(: 16) = (/ 'so4_a1          ','bc_a4           ','SVOC            ','bc_a1           ','CO              ', &
+      extfrc_lst(: 24) = (/ 'so4_a1          ','bc_a4           ','SVOC            ','bc_a1           ','CO              ', &
```

Comment:
The indexing is wrong here. There are only 16 elements. Similar indexing errors also appear below and in other `mo_sim_dat.F90`

Reply:
Thanks for catching that. I manually reverted the changes and forgot that index.